### PR TITLE
Disquse find

### PR DESCRIPTION
**Pull request by PichotM**
Document `SET_VEHICLE_MAX_SPEED `and `IS__VEHICLE_A_LECTRO`

### CODE REVIEW

File: VEHICLE/SetVehicleMaxSpeed.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+aliases: ["0xBAA045B4E42F3C06"]
+---
+## _SET_VEHICLE_MAX_SPEED
+
+```c
+// 0xBAA045B4E42F3C06
+void _SET_VEHICLE_MAX_SPEED(Vehicle vehicle, float speed);
+```
+
+```
```
Comment (stannum-cfx) [inline VEHICLE/SetVehicleMaxSpeed.md]: Please remove the code block from the description. It's a descriptive message about the native, it's not a code snippet ;)

Comment (PichotM) [reply VEHICLE/SetVehicleMaxSpeed.md]: Done!

**Comment by Disquse** [comment]
@PichotM I found out that `_IS_VEHICLE_A_LECTRO` is a wrong name. There's another bike with KERS called `vindicator` for which this native will return `true` as well. So probably this should be named like `_IS_VEHICLE_HAVE_KERS_BOOST` or so.

**Comment by PichotM** [comment]
_HAS_VEHICLE_KERS_BOOST might be better
(_HAS_VEHICLE_ROCKET_BOOST / _HAS_VEHICLE_JUMPING_ABILITY also exist)

**Comment by Syntasu** [comment]
`_DOES_VEHICLE_HAVE_KERS ` seems more suiting, as we already have similar natives:
```
_DOES_VEHICLE_HAVE_DOOR
DOES_VEHICLE_HAVE_ROOF
DOES_VEHICLE_HAVE_STUCK_VEHICLE_CHECK
DOES_VEHICLE_HAVE_WEAPONS
_DOES_VEHICLE_HAVE_DECAL
```

**Comment by blattersturm** [comment]
Native almost certainly starts with `GET_VEHICLE_*`, would have to check a few console build patches by hand (boring!) to get a jhash.

**Comment by blattersturm** [comment]
... that was easy, just listed all unknown jhashes from latest last-gen build (since this native existed there), `get_vehicle_has_kers` -> `0761e635` is in there

**Pull request merged**